src/styles/main.scss

```
@@ -493,9 +489,9 @@ input.disabled, select.disabled {
 
 // Use HTML5 input field for file upload
 .file-upload > input[type="file"] {
-  width: inherit !important;
+  width: 300px;
```

```suggestion
  width: 100%;
```
How about percentage width? Allows us to use more of the available space for displaying file name if file names become somewhat long. Also should not cause any overflow.

---

Fixed